src/plugins/shared_ux/public/components/index.ts

```diff
@@ -19,13 +19,21 @@ export const LazyExitFullScreenButton = React.lazy(() =>
   }))
 );
 
+export const LazySolutionToolbarButton = React.lazy(() =>
+  import('./toolbar/index').then(({ SolutionToolbarButton }) => ({
+    default: SolutionToolbarButton,
+  }))
+);
+
 /**
  * A `ExitFullScreenButton` component that is wrapped by the `withSuspense` HOC.  This component can
  * be used directly by consumers and will load the `LazyExitFullScreenButton` component lazily with
  * a predefined fallback and error boundary.
  */
 export const ExitFullScreenButton = withSuspense(LazyExitFullScreenButton);
 
+export const SolutionToolbarButton = withSuspense(LazySolutionToolbarButton);
```

Comment:
Could you add a comment here?

Reply:
yes thanks for catching that [e2c59b3](https://github.com/elastic/kibana/pull/125998/commits/e2c59b3d528eb48fbaf7241f3b8576ad759c0fcd) for changes